### ISSUE TITLE
Bottom bar should be not displayed in settings

### ISSUE THREAD

**Issue by kelson42** [kiwix/kiwix-js-windows#43]
The buttons do not not work if settings are displayed and it does not make sense here.

**Comment by Jaifroid** [comment]
Thanks, @kelson42 . This should probably be applied in Kiwix JS if @mossroy agrees and then ported to Kiwix JS Windows, since the code is common to both apps.

**Comment by kelson42** [comment]
@Jaifroid Good. I'm not always sure where the problem should be fixed, so in that case I used to report it where I see it first. If you want you can move tickets easily to kiwix-js with https://github-issue-mover.appspot.com/.

**Comment by Jaifroid** [comment]
So, I've got rid of the bottom toolbar when the Settings and the About tabs are open.

There is one navigation issue that arises with this: the text on the About page is quite long, and the up-arrow on the bottom toolbar can be used to navigate quickly back to the top. I'm not sure how important this is, as on a phone it's not hard to scroll rapidly back up, and on a non-touchscreen PC, Ctrl-Home or PgUp will take you back quickly. However, I thought I should flag it here, as it may have implications for https://github.com/kiwix/kiwix-js/issues/337 (@mossroy).

The other navigation issue is that the home icon is a quick shortcut for going back to the landing page. Clicking the Kiwix / Wikivoyage / WikiMed icon on the top toolbar does the same thing, but it is less discoverable. I've mitigated against this by adding a couple of Return to PageName... links on the About page (there were already a couple on the Settings page) -- see screenshot below. These Return to... links were always a kludge to provide a way of returning to the currently loaded article without having to return to the landing page and search for it again. This is a known issue in https://github.com/kiwix/kiwix-js/issues/127.

Basically I think there's a trade-off between more screen real estate and ease of navigation. However, I think I'm going to implement this in Kiwix JS Windows 0.9.9 beta and see if people squeal....

![image](https://user-images.githubusercontent.com/4304337/35777385-e1c3eb54-09a4-11e8-85a2-407a9911e6ea.png)

**Comment by kelson42** [comment]
@Jaifroid 

I do not think the lack of "back to top" is a real problem. In worse case, if its, a "back to top" bottom bar similar to the one on Android could be build IMO... But again, I would be really surprise if that becomes a recurrent feature requests in user's feedbacks.

Regarding the ability to come back to the content, I think implementing https://github.com/kiwix/kiwix-js-windows/issues/44 would also help to get that problem fixed.

**Comment by Jaifroid** [comment]
This issue is now implemented in 0.99 in Kiwix JS and in WikiMed (both versions in the Store), but not yet in Wikivoyage (update is in code, but not yet shipped).

I'm not entirely happy with the ergonomics, as returning to the article content from the Config or About pages requires two-handed use on large mobile screens when the bottom bar is hidden. But this can be discussed in https://github.com/kiwix/kiwix-js/issues/337 . I'm closing this issue.
